lib/Conch/Controller/User.pm

```diff
@@ -363,10 +363,9 @@ sub list ($c) {
 
 Creates a user. System admin only.
 
-Optionally takes a query parameters:
+Optionally takes a query parameter:
 
 * 'send_mail' (defaulting to true), to send an email to the user with the new password
```

Comment:
Should send_mail be pulled into the post body as well?

Reply:
IMO no, because it's not a value being set in the db but rather an out of band thing to signal something else to do at the same time. but I'll defer to our REST expert on this :)

Reply:
Since this is a POST only endpoint it REST doesn't have an opinion here. Either is acceptable.